.circleci/config.yml

```diff
@@ -178,15 +178,22 @@ jobs:
       - run:
           name: "Test Modified Packages"
           command: |
-            PACKAGES=$(node tooling/modified-packages.js)
+            PACKAGES=$(node tooling/modified-packages.js --single-line)
 
             echo "Modified Packages:"
             echo "${PACKAGES}"
-
+            
+            VAR1="@webex/plugin-meetings"
```

Comment:
This does look hacky , may be we can refactor this later 

@InteractiveTimmy any thoughts ?

Reply:
This should be fine for now [as long as it continues to operate CI normally]. We'll have independent building once we get through the tooling elevation tasks.